src/csmacnz.Coveralls/CoverallData.cs

```diff
@@ -13,6 +13,9 @@ public sealed class CoverallData
         [JsonProperty("service_name")]
         public string ServiceName { get; set; }
 
+        [JsonProperty("service_number")]
```

Comment:
, DefaultValueHandling = DefaultValueHandling.Ignore 
because this is listed as optional in the api.